tests/db_utils.ts

```
@@ -0,0 +1,110 @@
+    // SQLite in-memory is cleared on disconnect/reconnect, or file can be deleted if file-based.
+    // For :memory:, typically no explicit DROP TABLES is needed as it's fresh.
+    console.log('SQLite in-memory, skipping explicit clear.');
+    return;
+  }
+
+  if (dbType === 'mysql') {
+    let connection: MySqlConnection | null = null;
+    try {
+      connection = await mysql.createConnection(connectionString);
+      const tables = await getDbQueryResults<{ [key: string]: string }>(connection, 'SHOW TABLES');
+      const tables = await getDbQueryResults<{ [key: string]: string }>(connection, 'SHOW TABLES');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'tables'. Consider to delete it or rename it.

'tables' is defined here:


(lint/suspicious/noRedeclare)